ts/navigation/AppStackNavigator.tsx

```diff
@@ -196,7 +196,13 @@ const InnerNavigationContainer = (props: { children: React.ReactElement }) => {
         [ROUTES.SERVICES_NAVIGATOR]: {
           path: "services",
           screens: {
-            [ROUTES.SERVICE_DETAIL]: "service-detail",
+            [ROUTES.SERVICE_DETAIL]: {
+              path: "service-detail",
+              parse: {
+                serviceId: String,
+                activate: Boolean
```

Comment:
```suggestion
                activate: value => value === "true"
```

The `Boolean` constructor won't actually parse the string value given: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/Boolean#creating_boolean_objects_with_an_initial_value_of_true. Does this change make sense to you?

Reply:
Yeah, I've already made it actually [dcaecb6](https://github.com/pagopa/io-app/pull/4085/commits/dcaecb607d8b593d74d061ef0bc3afe249b38cfd)